sdk/devcenter/ci.yml

```diff
@@ -0,0 +1,31 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+
+trigger:
+  branches:
+    include:
+    - main
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/devcenter/
+
+pr:
+  branches:
+    include:
+    - main
+    - feature/*
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/devcenter/
```

Comment:
Can we split dataplane and management plane pipelines?

Please refer [this](https://github.com/Azure/azure-sdk-for-net/pull/31589/files) PR to see how to split it.

Reply:
Added the filter on paths here, as well as the SDKType param